.github/workflows/release.yml

```diff
@@ -103,9 +103,40 @@ jobs:
       with:
         name: ${{ steps.get_version.outputs.VERSION }}
         path: test_wheel/emcache-*.whl
+    
+  build_linux_aarch64:
+    outputs:
+      version: ${{ steps.get_version.outputs.VERSION }}
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.11"
+    - name: Generate manylinux2014_aarch64-wheel
+      run: |
+        brew install docker
+        brew install colima
+        colima start
+        docker run --platform linux/arm64 -v `pwd`:/io -e "PYTHON_VERSION=3.11" quay.io/pypa/manylinux2014_aarch64 /io/bin/build_manylinux1_wheel.sh
```

Comment:
shouldnt we provide the Python version from the matrix  as we do here https://github.com/emcache/emcache/pull/94/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R80?

IIRC this is the way that we generate the different releases for the different supported Python versions

Reply:
ahh didn't realise that it was building per python version, will update